examples/react/basic/src/main.tsx

```diff
@@ -82,6 +86,179 @@ function IndexComponent() {
   )
 }
 
+export const anchorRoute = createRoute({
```

Comment:
move this into `e2e/react-router/basic-file-based` and add a new `scrollIntoView.spec.ts` test (test name just a proposal)

Reply:
Done, thanks!